client/src/commons/LocationInputField/LocationInput.tsx

```diff
@@ -73,48 +74,27 @@ const LocationInput = (props: LocationInputProps) => {
         _isMounted && setSelectedAddress(geoCodedAddress as GoogleApiPlace);
     };
 
+    const AutocompleteComponent =
+        <AutocompletedField
+            required={required}
```

Comment:
require  attribute behaviour is conflicting our form submit design.
in chrome browser it is not possible to submit form while there is unfinished required fields, so the browser wont let you fire submit event (aka - save the data of the current tab and move to other tab) if there is required field. 
this validation checked in the schema instead.
if you pass false to the require prop there is no problem but if you pass yes you wont be able to move tabs while the field is empty (unwanted behaviour) i suggest to remove this prop

Reply:
sure, removed. thanks